src/EventGrid/EventGrid/Models/PSDomain.cs

```diff
@@ -30,6 +30,80 @@ public PSDomain(Domain domain)
             this.ProvisioningState = domain.ProvisioningState;
             this.Tags = domain.Tags;
             this.Endpoint = domain.Endpoint;
+            this.InputSchema = domain.InputSchema;
```

Comment:
is it necessary to have domain?.InputSchema

Reply:
Not really since domain is an input parameter and we are not calling this method/class unless we need to display an actual value.